modules/ROOT/pages/rest-api-access.adoc

```diff
@@ -12,28 +12,42 @@ include::partial$_std-hdr-sgw.adoc[]
 include::partial$block-abstract.adoc[]
 
 :xref--pfx: {sgw--xref}:
+:tls-encryption--xref: {sgw--xref}{secure-sgw-access--page}[enforcing TLS encryption]
 
 == Overview
 
-In addition to the {rest-api--xref}, Sync Gateway provides secure access to two REST API  targeted at specific user populations and functional areas, namely the:
+Sync Gateway provides secure access to its REST APIs, namely the:
 
-* {rest-api-admin--xref}
-* {rest-api-metrics--xref}
+* {rest-api--xref}
+* {rest-api-admin--xref} -- for the administration and configuration of {sgw}
+* {rest-api-metrics--xref} -- for the monitoring of {sgw} performance metrics
 
 Each REST API is accessed through a different, <<lbl-port-cfg, user-specifiable>>, TCP port.
-This makes it easy to control their physical exposure, perhaps keeping the Admin REST API secure behind your firewall.
+This makes it easy to control their physical exposure, perhaps to keep the Admin REST API secure behind your firewall.
 
-{sgw-s} 3.0 brings additional layers of security to the Metrics and Admin interfaces by requiring Couchbase Server Role-Based Access Control (RBAC) authentication for all <<lbl-secure-users,secure API users>>, and by {sgw--xref}{security--page}[enforcing TLS encryption].
+{sgw-s} 3.0 brings additional layers of security by {tls-encryption--xref} for all API traffic and by requiring Couchbase Server Role-Based Access Control (RBAC) authentication for all <<lbl-secure-users,Admin and Metrics API users>>.
```

Comment:
On the staging site {sgw-s} is just that in plain text. Not sure if thats just because its in staging or if there is an issue with it.

Reply:
Additionally I know you specify later but might be worth noting that TLS isn't really "enforced". Its enabled by default but a customer can turn it off. Same with the Admin Auth, its not 'required' its optional on the customers part.

Reply:
> On the staging site {sgw-s} is just that in plain text. Not sure if thats just because its in staging or if there is an issue with it.

There is a staging site issue causing these shared variables not to resolve.